Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -120,7 +119,9 @@ private Furniture(Furniture other)
 
         Parameters = new Parameter(other.Parameters);
         Jobs = new BuildableJobs(this, other.Jobs);
-        workshop = other.workshop; // don't need to clone here, as all are prototype things (not changing)
+
+        // don't need to clone here, as all are prototype things (not changing)
```

Comment:
Actually I would propose to use Params as only thing that can change.. this would be a generic way how to access parameters from other components, easy to mod, and would also solve the persistency (save/load game). So basically no more prototype vs. instance.. only difference will be that instance will have 'live' Params